etc/docker/elyra_development/Dockerfile

```diff
@@ -21,8 +21,10 @@ RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-4.5.11-Linux-x86
 
 # update nodejs
 RUN curl -fsSL https://deb.nodesource.com/setup_16.x | bash - && \
-    apt-get install -y nodejs && \
-    rm setup_16.x
+    apt-get install -y nodejs
+
+# cleanup node files
+RUN rm -rf setup_16.x
```

Comment:
https://github.com/elyra-ai/elyra/issues/2859#issuecomment-1209064413 should still apply since no changes were made to the `Dockerfile` in this PR that would avoid the issue

Reply:
I see.  Looks like there's _no_ output file via this command sequence so `RUN rm -rf setup_16.x` (or `rm setup_16.x` in the previous version) should be removed entirely.